Cargo.toml

```diff
@@ -2,6 +2,16 @@
 name = "gevulot-rs"
 version = "0.1.0"
```

Comment:
Do we go with 0.1.0 or should we bump this e.g. to 0.1.3 at least to match **ember**?

Reply:
I don't know, should we?

Reply:
However this can be our public version of the network, so probably it makes sense

Reply:
Done